themes/src/main/resources/theme/keycloak/welcome/theme.properties

```
@@ -5,3 +5,5 @@ styles=css/welcome.css
 
 # When set to true, the user will be redirected to the Administration Console if an administrative users already exists.
 redirectToAdmin=true
+kcDarkModeClass=pf-v5-theme-dark
+kcDarkModeSwitchEnabled=true
```

Same here, we can drop the prefix for this one.

```suggestion
darkModeEnabled=true
```

---

that means that this theme is wrong

---

also I could remove the `kcDarkModeClass` completely as all the classes are hardcoded anyway

---

Not really, there is a similar property right above it (`redirectToAdmin`). The main reason why the prefix exists is for the CSS class names, which can be a LOT.